Makefile

```diff
@@ -154,15 +168,17 @@ $(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies/bower
 	| $(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-dependencies
 	./scripts/build/dependencies-bower.sh -p -x $|
 
-$(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(TARGET_ARCH)-app: \
+$(BUILD_DIRECTORY)/electron-$(TARGET_PLATFORM)-$(APPLICATION_VERSION)-$(TARGET_ARCH)-app: \
```

Comment:
> every time you do a make after changing commits, you'll end up with an entirely separate directory, which kinda destroys the artefact-caching that Make is designed to give you

Well, if you change commits then the actual package contents is supposed to have change, so I guess that's intended?

> I'll have a play and see what I come up with...

Looking forward to that!